xmtp_db/src/encrypted_store/group.rs

```diff
@@ -554,6 +554,33 @@ impl DbConnection {
 
         Ok(())
     }
+
+    pub fn find_duplicate_dms_for_group(
```

Comment:
Just to put on your radar, something very similar [exists](https://github.com/xmtp/libxmtp/blob/7cefaa267df9970ab4ff471b1eaf2bcfdecb745f/xmtp_db/src/encrypted_store/group/dms.rs#L45-L66), the only difference is it excludes the id of the group you provided.